docs/user_guide/architecture.md

```diff
@@ -657,6 +657,8 @@ procedure that involves multiple models, such as "data preprocessing
 -> inference -> data postprocessing".  Using ensemble models for this
 purpose can avoid the overhead of transferring intermediate tensors
 and minimize the number of requests that must be sent to Triton.
+Want to try out ensembles? Checkout 
+[this guide](https://github.com/triton-inference-server/tutorials/tree/main/Conceptual_Guide/Part_5-Model_Ensembles).
```

Comment:
Move this to be first in the list of ensemble examples at the bottom of this section: https://github.com/triton-inference-server/server/blob/main/docs/user_guide/architecture.md#additional-resources
```suggestion
```